downstream/modules/platform/proc-creating-the-custom-execution-environment-definition.adoc

```diff
@@ -0,0 +1,161 @@
+//Used in downstream/titles/aap-installation-guide/platform/assembly-disconnected-installation.adoc
+
+:_newdoc-version: 2.15.1
+:_template-generated: 2024-02-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-the-custom-execution-environment-definition_{context}"]
+= Creating the custom {ExecEnvShort} definition
+
+[role="_abstract"]
+
+Once the {Builder} RPM is installed, use the following steps to create your custom {ExecEnvShort}.
+
+. Create a directory for the build artifacts used when creating your custom {ExecEnvShort}.	Any new files created with the steps below will be created under this directory.
++
+----
+$ mkdir $HOME/custom-ee $HOME/custom-ee/files
+$ cd $HOME/custom-ee/
+----
++
+
+. Create an `execution-environment.yml` file that defines the requirements for your custom {ExecEnvShort}. 
+
+.. Override the base image to point to the minimal execution environment available in your {PrivateHubName}.
+
+.. Define the additional build files needed to point to any disconnected content sources that will be used in the build process.
+Your custom `execution-environment.yml` file should look similar to the following example:
+
++
+----
+$ cat execution-environment.yml
+---
+version: 3
+
+images:
+  base_image:
+    name: private-hub.example.com/ee-minimal-rhel8:latest
+
+dependencies:
+  python: requirements.txt
+  galaxy: requirements.yml
+
+additional_build_files:
+  - src: files/ansible.cfg
+    dest: configs
+  - src: files/pip.conf
+    dest: configs
+  - src: files/hub-ca.crt
+    dest: configs
+  # uncomment if custom RPM repositories are required
+  #- src: files/custom.repo
+  #  dest: configs
+
+additional_build_steps:
+  prepend_base:
+    # copy a custom pip.conf to override the location of the PyPI content
+    - ADD _build/configs/pip.conf /etc/pip.conf
+    # remove the default UBI repository definition
+    - RUN rm -f /etc/yum.repos.d/ubi.repo
+    # copy the hub CA certificate and update the trust store
+    - ADD _build/configs/hub-ca.crt /etc/pki/ca-trust/source/anchors
+    - RUN update-ca-trust
+    # if needed, uncomment to add a custom RPM repository configuration
+    #- ADD _build/configs/custom.repo /etc/yum.repos.d/custom.repo
+
+  prepend_galaxy:
+    - ADD _build/configs/ansible.cfg ~/.ansible.cfg
+
+...
+----
++
+
+. Create an `ansible.cfg` file under the `files/` subdirectory that points to your {PrivateHubName}, and contains the token used to authenticate to the hub and download collections.
```

Comment:
the token is not located in this file but in an environment variable set in a subsequent section, so everything after the comma can probably be removed.

i know QE ran through this once but i want to triple-check that having the token in the environment variable works as expected before publishing.